code/go/pkg/validator/validator_test.go

```diff
@@ -132,6 +132,12 @@ func TestValidateFile(t *testing.T) {
 				"field streams.0.vars.3: Must not be present",
 			},
 		},
+		"bad_skip_ignored_fields": {
```

Comment:
This package seems to have more issues, did you create it by copying an old package? Consider creating it from scratch with `elastic-package create package` and `elastic-package create data-stream`.